Source/WebCore/page/Quirks.cpp

```diff
@@ -365,6 +365,16 @@ bool Quirks::shouldAvoidUsingIOS13ForGmail() const
 #endif
 }
 
+// mail.google.com rdar://128360054
+// FIXME (rdar://130624461): Remove this quirk once Gmail adopts the `writingsuggestions` attribute.
+bool Quirks::shouldDisableWritingSuggestionsByDefault() const
+{
+    if (!needsQuirks())
+        return false;
+    auto& url = m_document->topDocument().url();
+    return url.host() == "mail.google.com"_s;
```

Comment:
Is Gmail always at this domain? I was wondering if there was some variation where corporations used their own domain.

Reply:
Not certain... this does match the other Gmail quirks in this file though.